### ISSUE TITLE
Emotion Analysis Of Conversantion/Waifu algorithm idea?

### ISSUE THREAD

**Issue by HRNPH** [HRNPH/AIwaifu#17]
for now, I'm thinking of training sentiment classification on multiple classes we will then use the top Log(probability) on each neuron as a combination of emotions ['Joy', 'Mad'] the model that I currently use for prototyping contains a little emotion but due to the complex emotion that can be expressed in the real conversation such as Tsundere(actually happy but speaking as if they're mad: *BLUSHING*  I hate you! (Maybe recognizable if you took a whole conversation as features) 
this issue was opened for the suggestion of a better Conversation -> Emotion algorithm

**Comment by andri-jpg** [comment]
I think sentiment classification is quite good because it also doesn't take too much memory, the challenge might be to label the data set with more classes like shy, tsundere and many other expressions.

**Comment by HRNPH** [comment]
yeah i think maybe I need to make some datasets myself for fine tuning
